selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -57,6 +59,15 @@ def __init__(self, CP, init_v=0.0, init_a=0.0):
     self.a_desired_trajectory = np.zeros(CONTROL_N)
     self.j_desired_trajectory = np.zeros(CONTROL_N)
     self.solverExecutionTime = 0.0
+    self.params = Params()
+    self.param_read_counter = 0
+    self.read_param()
+    self.personality = log.LongitudinalPersonality.standard
+
+  def read_param(self):
+    param_value = self.params.get('LongitudinalPersonality')
+    if param_value is not None:
```

Comment:
probably want an else to put this in a defined state if the param exists then gets deleted